Dockerfile

```diff
@@ -483,9 +486,9 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     && sudo rm -rf /var/lib/apt/lists/*
 
 # Python
-ENV PYENV_ROOT=/opt/python
-ENV PATH=$PATH:$PYENV_ROOT/shims:$PYENV_ROOT/bin:$PYENV_ROOT/conan2/bin
-COPY --from=python --chown=$USER:$USER $PYENV_ROOT $PYENV_ROOT
+ENV PYTHON_INSTALL_ROOT=/opt/python
+ENV PATH=$PATH:$PYTHON_INSTALL_ROOT/shims:$PYTHON_INSTALL_ROOT/bin:$PYTHON_INSTALL_ROOT/conan2/bin
+COPY --from=python --chown=$USER:$USER $PYTHON_INSTALL_ROOT $PYTHON_INSTALL_ROOT
 
 # NodeJS
 ENV NVM_DIR=/opt/nvm
```

Comment:
It looks like Docker couldn't find NODEJS_VERSION here so I had to add
`ARG NODEJS_VERSION`